yt_dlp/extractor/youtube.py

```diff
@@ -348,6 +348,16 @@ class YoutubeBaseInfoExtractor(InfoExtractor):
         r'(?:www\.)?kbjggqkzv65ivcqj6bumvp337z6264huv5kpkwuv6gu5yjiskvan7fad\.onion',
         r'(?:www\.)?grwp24hodrefzvjjuccrkw3mjq4tzhaaq32amf33dzpmuxe7ilepcmad\.onion',
         r'(?:www\.)?hpniueoejy4opn7bc4ftgazyqjoeqwlvh2uiku2xqku6zpoa4bf5ruid\.onion',
+        # piped instances
```

Comment:
```suggestion
        # piped instances from https://github.com/TeamPiped/Piped/wiki/Instances
```

Make sure to add all these

Reply:
Added the ones that work (piped.notyourcomputer.net seems to redirect to the github and not an instance)